elasticai/creator/vhdl/language.py

```diff
@@ -180,6 +270,11 @@ def _add_semicolons(lines: Code) -> Code:
     yield f"{temp[-1]}"
 
 
+def _add_semicolon_in_each_line(lines: Code) -> Code:
```

Comment:
just thought about renaming this to `def _append_semicolons_to_lines(lines: Code) -> Code:`